sdk/identity/identity/src/credentials/managedIdentityCredential/index.ts

```diff
@@ -14,9 +14,20 @@ import { imdsMsi } from "./imdsMsi";
 import { MSI } from "./models";
 import { appServiceMsi2017 } from "./appServiceMsi2017";
 import { arcMsi } from "./arcMsi";
+import { processMultiTenantRequest } from "../../util/validateMultiTenant";
 
 const logger = credentialLogger("ManagedIdentityCredential");
 
+/**
+ * Options for the {@link ManagedIdentityCredential}
+ */
+export interface ManagedIdentityCredentialOptions extends TokenCredentialOptions {
+  /**
+   * Allows specifying a tenant ID
+   */
+  tenantId?: string;
```

Comment:
We're not adding a `tenantId`  option to `ManagedIdentityCredentialOptions` this should be removed.

Reply:
Thank you!